VstsTasks/AzureDtlCreateVM/task-funcs.ps1

```diff
@@ -0,0 +1,180 @@
+function Handle-LastError

```

Comment:
Unfortunately, I cannot do anything about that as the task expects all its files to be relative, within the folder structure.